bentoml/_internal/utils/tensorflow.py

```diff
@@ -1,14 +1,17 @@
-import logging
+import importlib.util
```

Comment:
Also I should fix this, we shouldn't rely on this https://github.com/asottile/scratch/wiki/ImportSideEffects

Reply:
However, autoflake `--remove-unused-imports` seems to rely on this behaviour

Reply:
we don't use autoflake but there might be scenario where developers use it. Put it here for future reference

Reply:
does it help if we move this to the get_tf_version function below?